src/foam/nanos/dig/DUGRuleAction.js

```diff
@@ -69,9 +74,23 @@
         DUGDigestConfig dugDigestConfig = (DUGDigestConfig) dugDigestConfigDAO.find(rule.getSpid());
         DUGRule dugRule = (DUGRule) rule;
         AbstractSink sink = null;
-        if ( dugDigestConfig != null && dugDigestConfig.getEnabled() ) {
+        String url = getUrl();
```

Comment:
Another option here is to create DUGLoopbackRuleAction which extends DUGRuleAction and overrides getUrl() and then the rule action is set to this.